INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:app:Status request successfull OK - healthy
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:13:37] "[37mGET /healthz HTTP/1.1[0m" 200 -
INFO:app:Post id : 12 retrieved
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:13:40] "[37mGET /12 HTTP/1.1[0m" 200 -
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:13:46] "[37mGET / HTTP/1.1[0m" 200 -
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:13:51] "[37mGET /create HTTP/1.1[0m" 200 -
INFO:app:New Post : 13 created
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:14:04] "[32mPOST /create HTTP/1.1[0m" 302 -
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:14:04] "[37mGET / HTTP/1.1[0m" 200 -
ERROR:app:Post id : 14 missing
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:14:10] "[33mGET /14 HTTP/1.1[0m" 404 -
INFO:app:Post id : 13 retrieved
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:14:14] "[37mGET /13 HTTP/1.1[0m" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
CRITICAL:app:DB MISSING : check DB file:tevvchtrends.db?mode=rw exists
ERROR:app:Exception on /13 [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 46, in post
    post = get_post(post_id)
  File "app.py", line 27, in get_post
    post = connection.execute('SELECT * FROM posts WHERE id = ?',
AttributeError: 'NoneType' object has no attribute 'execute'
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:15:57] "[1m[35mGET /13 HTTP/1.1[0m" 500 -
INFO:app:About Us Page retrieved
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:16:04] "[37mGET /about HTTP/1.1[0m" 200 -
INFO:app:Status request successfull : {'db_connection_count': 0, 'post_count': 0}
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:16:08] "[37mGET /metrics HTTP/1.1[0m" 200 -
CRITICAL:app:DB MISSING : check DB file:tevvchtrends.db?mode=rw exists
INFO:app:Status request successfull ERROR - unhealthy
INFO:werkzeug:127.0.0.1 - - [28/Aug/2021 15:16:12] "[1m[35mGET /healthz HTTP/1.1[0m" 500 -
